### PR TITLE
added single video page

### PR DESCRIPTION
**Pull request by saicharan-gouru**
-Created Single video page
 * Added video iframe.
 * Added video details (description, views, creator details).
 * Fetched and displayed similar videos.
single video-min (1)

### CODE REVIEW

File: src/pages/SingleVideoPage/SingleVideoPage.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import {useData} from "../../contexts";
+import {useParams,Link} from "react-router-dom";
+
+function SingleVideoPage(){
+
+    const { videos } = useData();
+    const { _id } = useParams();
+    const getSingleVideo = videos.filter((item) => item._id===_id);
+    const getSimilarVideos = videos.filter((item) => item.categoryName === getSingleVideo[0].categoryName);
+    console.log(getSimilarVideos)
```
Comment (junaidshaikh-js) [inline src/pages/SingleVideoPage/SingleVideoPage.js]: remove console logs

File: src/pages/Videoslisting/Videolisting.js [hunk 2]
```diff
@@ -15,8 +16,9 @@ function Videolisting(){
                 <button className="filter-btn" onClick={() => videosDispatch({type:"FILTER",payload:"Cartoon"})}>Cartoon</button>
             </div>
             <div className="videos-container">
-                {filteredVideos.map(item => 
-                <div className="video-card">
+                {filteredVideos.map( 
+                item =>
+                <Link to={`/video/${item._id}`} className="video-card">
```
Comment (junaidshaikh-js) [inline src/pages/Videoslisting/Videolisting.js]: add `key` attribute

**Review by junaidshaikh-js — APPROVED** [review]
Good work! 

Suggested some changes.

**Review by anupkgurung — APPROVED** [review]
code is good,
good to merge.

**Pull request merged**